databricks/koalas/series.py

```diff
@@ -348,6 +348,25 @@ def index(self):
             raise KeyError('Currently supported only when the Column has a single index.')
         return self._kdf.index
 
+    @property
+    def is_unique(self):
+        """
+        Return boolean if values in the object are unique
+
+        .. note:: this is an expensive operation that has to count all values in the Series.
```

Comment:
i don't know if you need this note here ... of course it has to count all the values :)